pkgs/applications/blockchains/bitcoin/default.nix

```diff
@@ -24,23 +24,21 @@
 
 with lib;
 let
-  version = "0.21.1";
-  majorMinorVersion = versions.majorMinor version;
   desktop = fetchurl {
-    url = "https://raw.githubusercontent.com/bitcoin-core/packaging/${majorMinorVersion}/debian/bitcoin-qt.desktop";
+    url = "https://raw.githubusercontent.com/bitcoin-core/packaging/aacdf188e340c8292c8eae3f634eec6aa0dc3244/debian/bitcoin-qt.desktop";
```

Comment:
We should use the version here to make sure we do not use a really old file.

Reply:
Well, the upstream has stopped versioning stuff, see https://github.com/bitcoin-core/packaging/issues/70

Reply:
There is still versioning, but it will only be done manually for the snap package, usually a few days after the release.

If this is too slow for you, an alternative would be to use the version from master?

Reply:
I think that's still fine. I see you just created `22.x` branch and we'll use this. Thanks!

Reply:
Addressed in 9cdc7e7ae87f1410c139e56a46886f61676f339e